fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx

```diff
@@ -39,22 +39,38 @@ const ViewSingleRollCall = () => {
   }
 
   const laoId = RollCallHooks.useAssertCurrentLaoId();
+  const isConnected = RollCallHooks.useConnectedToLao();
 
   switch (rollCall.status) {
     case RollCallStatus.CREATED:
-      return <RollCallCreated rollCall={rollCall} laoId={laoId} isOrganizer={isOrganizer} />;
+      return (
+        <RollCallCreated
+          rollCall={rollCall}
+          laoId={laoId}
+          isConnected={isConnected}
+          isOrganizer={isOrganizer}
+        />
+      );
     case RollCallStatus.REOPENED:
     case RollCallStatus.OPENED:
       return (
         <RollCallOpen
           rollCall={rollCall}
           laoId={laoId}
+          isConnected={isConnected}
```

Comment:
should this be a parameter? shouldn't we just use the hook within those components? 🤔

Reply:
That is a good question but also applies to the other props here, especially laoId. All of them can be retrieved from the global application state but we can also pass them using props. At the time I am writing this I do not see a big advantage for either way. What do you think?

Reply:
In terms of examples in the codebase, it might make sense to use the hook for `isConnected`. While the other params make sense for a roll call (conceptually), `isConnected` feels wrong in the API, it shouldn't be an external concern for a roll call.

Actually, I was even thinking, should we have the `isConnected` hook (almost) everywhere or could we put it in a global context ?

Reply:
It kind of already is, it is in the feature context provided by the lao feature. Putting it in a separate context comes with the question where. Only the lao feature knows how to retrieve this information from redux. Hence we would have to put it in the LaoNavigator component wrapping all lao screens and export the context from the lao feature and all other features would then need to import the context. If we only put the `isConnected` hook there this seems to be a bit of an overkill but now when thinking about it, I think we could put other things in there too such as the laoId. For instance we could get rid of the `getCurrentLaoId` completely and would reduce the number of feature in- and exports :)

Reply:
Oh right, I forgot about that implementation aspect. Thanks for the reminder. I see the point you're making.

It does make sense, but at this stage, where the system as a whole is becoming pretty stable, I'm not sure we'd want to do yet another far-reaching change as opposed to stabilizing what we already have. What do you think?